03_launch_mgmt_cluster.sh

```diff
@@ -38,6 +38,10 @@ if [ "${CAPI_VERSION}" == "v1alpha2" ]; then
   CAPBMBRANCH="${CAPBMBRANCH:-master}"
 elif [ "${CAPI_VERSION}" == "v1alpha1" ]; then
   CAPBMBRANCH="${CAPBMBRANCH:-v1alpha1}"
+elif [ "${CAPI_VERSION}" == "v1alpha3" ]; then
+  CAPBMBRANCH="${CAPBMBRANCH:-v1alpha3}"
+  # v1alpha3 branch not yet in upstream. Once it is, below Nordix repo line is removed
+  CAPBMREPO="https://github.com/Nordix/cluster-api-provider-baremetal.git"
```

Comment:
I have a PR that takes care of that related to the branching

Reply:
Yep. Rebase and updates needed once that PR is merged.